### ISSUE TITLE
How to deal with (aggregated) measurements

### ISSUE THREAD

**Issue by rmarx** [quiclog/internet-drafts#139]
- placeholder issue so it can be linked in a qlog update email. Will flesh out later with details -

**Comment by LPardue** [comment]
This is interesting but I'm struggling to identify  clear or actionable feedback on the qlog draft. 

Robin said:

>  This all seems to indicate measurement-specific events is the way to go (i.e., not re-using existing qlog endpoint events), as it would be confusing to have those semantics in endpoint events.

If qlog is flexibile enough for their needs and we don't need to do anything, let's close the issue. If people don't agree, please speak up now.

**Comment by rmarx** [comment]
Agreed with Lucas here. There more I've thought about this over time, the more it seems to me this is potentially something perpendicular to qlog itself and/or can be done as a standalone extension later. 

Any indication on progress for this within spindump @jariarkko @ihlar? If not, we'll close this with no action soon.
